### PR TITLE
Edge Example App: CPU Temperature

### PR DESCRIPTION
**Pull request by ahmadateya**

### CODE REVIEW

File: _posts/2022-09-05-edge-example-app-cpu-temp.markdown [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+```log
+2022/09/04 11:11:25 {"Temp":43}
+2022/09/04 11:12:25 {"Temp":42.5}
+2022/09/04 12:13:25 {"Temp":42.5}
+2022/09/04 12:14:25 {"Temp":43}
+2022/09/04 13:15:25 {"Temp":44}
+```
+
+## Example of presented data
+<img src="/assets/images/cpu-temp-graph.png" alt="CPU graph line" width="800"/>
```
Comment (pkliczewski) [inline _posts/2022-09-05-edge-example-app-cpu-temp.markdown:38]: Please add image

Comment (ahmadateya) [reply _posts/2022-09-05-edge-example-app-cpu-temp.markdown]: done

**Review by pkliczewski — COMMENTED** [review]
Looks good to me. Please let me know when ready.

**Comment by pkliczewski** [comment]
@ahmadateya please fix `CheckLinks` issues

**Comment by ahmadateya** [comment]
the CheckLinks issue is related to the Part 1 and Part 2 articles because I reference them here, should I comment those lines for now and open another PR to it after merging Part 1 and Part 2 articles?

**Comment by pkliczewski** [comment]
If missing the other part is the only reason it is OK. We will merge both PRs at the same time.

**Review by pkliczewski — APPROVED** [review]

**Review by gciavarrini — APPROVED** [review]

**Comment by gciavarrini** [comment]
@ahmadateya I think the problem is that you use relative path (i.e. `/flotta/2022/09/05/edge-example-app-sense-the-internet-part-2.html`) instead of full link (i.e. `https://project-flotta.io/flotta/2022/09/04/edge-example-app-sense-the-internet-part-1.html`)
Can you please check and fix properly all the involved posts?

**Comment by gciavarrini** [comment]
@ahmadateya `CheckLinks` still failing, please double check all the links

**Comment by ahmadateya** [comment]
@gciavarrini I think the action run before merging #117, could we run it again?

**Pull request merged**